static/js/index.js

```diff
@@ -1292,9 +1295,77 @@ const hooks = {
         });
       }
     }
+    if (!context.callstack.docTextChanged) {
+      return cb();
+    }
+    const ace = context.editorInfo.editor;
+    const padlines = ace.exportText().split('\n');  
+    let i = 0;
+    const result = [];
+    if (padlines){
+      for (i = 0; i < padlines.length; i += 1){
```

Comment:
``+=`` is for concatenation not incrementing.  Do ``i++``

????

Reply:
yes Thank you!
correcting it